src/decideid/main.mo

```diff
@@ -20,6 +25,31 @@ shared ({ caller = deployer }) actor class DecideID(env : CommonTypes.ENV) = thi
     stable var _canistergeekLoggerUD : ?Canistergeek.LoggerUpgradeData = null;
     private let canistergeekLogger = Canistergeek.Logger();
 
+    
+    // -----------  Account  ----------
+    
+    // main account map, pk: decideid
+    stable var accountsStable : [(Types.DecideID, Types.Account)] = []; 
```

Comment:
Lets have one class respopnsible for that

Reply:
sorry, what do you mean by "one class" ? I am using `public class AccountManager` only.